doc/classes/AudioStreamPlayer.xml

```diff
@@ -72,7 +72,13 @@
 			If [code]true[/code], the playback is paused. You can resume it by setting [code]stream_paused[/code] to [code]false[/code].
 		</member>
 		<member name="volume_db" type="float" setter="set_volume_db" getter="get_volume_db" default="0.0">
-			Volume of sound, in dB.
+			Volume of sound, in dB. (To convert to dB, try [code]linear2db[/code].)
```

Comment:
```suggestion
			Volume of sound, in dB. If you have a linear scale value and wish to convert it to dB, use [method @GlobalScope.linear2db].
```

Reply:
Oh, wait, I didn't properly resolve this. The change was made in 49db65e2d928fceae93bd746e2194cdb8b01367d